python/ml4ir/applications/ranking/model/metrics/helpers/metrics_helper.py

```diff
@@ -66,12 +76,30 @@ def get_grouped_stats(
                 old_rank_col=old_rank_col,
                 new_rank_col=new_rank_col,
                 aux_label=aux_label,
-                group_keys=group_keys
+                group_keys=group_keys,
             ))
 
+    # Adding ranking metrics: MRR, ACR
+    df_clicked[RankingConstants.NEW_MRR] = 1.0 / df_clicked[old_rank_col]
+    df_clicked[RankingConstants.OLD_MRR] = 1.0 / df_clicked[new_rank_col]
+    df_clicked[RankingConstants.DIFF_MRR] = df_clicked[RankingConstants.NEW_MRR] - df_clicked[
+        RankingConstants.OLD_MRR]
+    df_clicked[RankingConstants.OLD_ACR] = df_clicked[old_rank_col]
+    df_clicked[RankingConstants.NEW_ACR] = df_clicked[new_rank_col]
+
     if group_keys:
+        # group df by group_keys
         df_grouped_batch = df_clicked.groupby(group_keys)
 
+        # get the power analysis metrics column names
```

Comment:
@mohazahran This filters it only to the ones specified in the eval config, correct?

Reply:
Yes